pkg/api/serve.go

```diff
@@ -29,6 +29,7 @@ import (
 
 const (
 	RequestIDHeaderName = "X-Request-ID"
+	SessionIDHeaderName = "X-Session-ID"
```

Comment:
Please document how this is different from Request-ID.  I _think_ that they do roughly the same thing.